packages/teleterm/package.json

```diff
@@ -18,6 +18,7 @@
     "build-renderer": "webpack build --config webpack.renderer.prod.config.js --progress",
     "build-natives": "electron-builder install-app-deps",
     "package": "electron-builder build --publish never -c.extraMetadata.name=teleconnect",
```

Comment:
Any opinions on the name? I think we should rename it to `teleport-connect` so that the command name is `teleport-connect` and not `teleconnect`.

Reply:
Agree, it should be called `teleport-connect`.

Reply:
One more thumbs up for `teleport-connect`